### ISSUE TITLE
Iniencephaly

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7365]
**Preferred term label:**
Iniencephaly

**Synonyms**

**Definition (free text, please give PubMed ID)**
Iniencephaly (IE) is a rare neural tube defect involving dysmorphic spine and abnormal fusion of the portion of the occipital skull with the back. It results in extreme fixed head retroflexion. This condition accounts for approximately 1% of all fetal malformations.

PMID:33294089
**Parent term (use hpo.jax.org/app)**
[Neural tube defect HP:0045005](https://hpo.jax.org/app/browse/term/HP:0045005)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
